README.md

```diff
@@ -75,85 +77,93 @@ salt us-central1-b n1-standard-1 10.240.136.204 123.45.67.89    RUNNING
 
 ## Software
 
-1. SSH to your Salt master and then become root
+1. SSH to your Salt master
     ```
     gcloud compute ssh salt --zone us-central1-b
-    sudo -i
+
     ```
+  1. Create a Compute Engine SSH key.
+  
+    If you do not have an existing key registered with Compute Engine, you will be prompted to create one.  This key will allow you to connect from your local workstation via SSH to any of your instances in your project.  You may use an Empty Passphrase but you should keep this key safe.  We will generate another key later in the installation to allow the Salt Master to talk to the Minions.  The document [Adding and Removing SSH keys](https://cloud.google.com/compute/docs/instances/adding-removing-ssh-keys#project-wide) describes SSH key management in more detail.
 
 1. Update packages and install dependencies
     ```
+    sudo -i
     apt-get update
     apt-get install python-pip git -y
     ```
 
-1. Install libcloud (v0.14.1 or greater)
+1. Install libcloud (v1.0.0-rc2 or greater)
```

Comment:
As of this writing, 1.0.0-rc2 is what's pulled in.  However, looking at the salt-cloud docs, the last version they reference is 0.14.0-dev (before they merged it with the salt repo), and their latest docs don't specify a version at all, just a pip install.

I changed it back to 0.14.1 as to not give the impression, as you received, that we've done something different that needs an updated libcloud version.